src/main/resources/application.yaml

```diff
@@ -45,6 +45,8 @@ spring:
       - authority-linking-rules-cache
     caffeine:
       spec: maximumSize=500,expireAfterWrite=3600s
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
Why is this needed?

Reply:
When to beans with the same name in one project, it overrides one on another. We have 2 UsersClient beans.

Reply:
I think we will have less problems if you just rename your client;

Reply:
So it can be removed as we use lib's bean?